capa/features/extractors/dnfile/helpers.py

```diff
@@ -186,7 +186,7 @@ def get_dotnet_managed_methods(pe: dnfile.dnPE) -> Iterator[DnType]:
         Each row represents a class in the current assembly.
             TypeName (index into String heap)
             TypeNamespace (index into String heap)
-            MethodList (index into MethodDef table; it marks the first of a continguous run of Methods owned by this Type)
+            MethodList (index into MethodDef table; it marks the first of a contiguous run of Methods owned by this Type)
```

Comment:
This is halfway between contiguous and continuous. Google Sheets guessed this and I have no idea/opinion, so I left it this way.

Reply:
lol, this one is funny